app/templates/_karma.conf.js

```diff
@@ -17,7 +17,10 @@ module.exports = function (config) {
 			'app/app.js',
 			'.generated/templates.js'<% if (useRouting) { %>,
 			'.generated/app/modules/routes/routes.concat.js'<% } %>,
-			'.generated/app/**/*.concat.js' // Leave this last!!!
+			'app/modules/**/*.module.js',
```

Comment:
But you lose the ability to guarantee unique variable names in your
components.
On Mon, Feb 8, 2016 at 7:43 PM John Woodruff <notifications@github.com>
wrote:

> In app/templates/_karma.conf.js
> <https://github.com/erikdonohoo/generator-ng-gulp/pull/4#discussion_r52261685>
> :
>
> > @@ -17,7 +17,10 @@ module.exports = function (config) {
> >  			'app/app.js',
> >  			'.generated/templates.js'<% if (useRouting) { %>,
> >  			'.generated/app/modules/routes/routes.concat.js'<% } %>,
> > -			'.generated/app/**/*.concat.js' // Leave this last!!!
> > +			'app/modules/**/*.module.js',
>
> There are some idiosyncrasies that happen when trying to test files with
> dependencies on other libraries and often make all your tests fail
> completely. I came across these issues again while adding support for
> component testing. It made them completely fail. We've encountered this on
> many occasions at the MTC and so rather than using the concatenated tests
> we use this in our karma configuration and it all works properly. Doesn't
> lose any functionality that we had before with the debugging
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/erikdonohoo/generator-ng-gulp/pull/4/files#r52261685>.
>